diff --git a/node_modules/expo-blur/ios/BlurEffectView.swift b/node_modules/expo-blur/ios/BlurEffectView.swift
index 185f69a..8467798 100644
--- a/node_modules/expo-blur/ios/BlurEffectView.swift
+++ b/node_modules/expo-blur/ios/BlurEffectView.swift
@@ -16,7 +16,11 @@ final class BlurEffectView: UIVisualEffectView {
 
   var tint: TintStyle = .light {
     didSet {
-      visualEffect = UIBlurEffect(style: tint.toBlurEffect())
+      if #available(iOS 26.0, *), tint == .glass {
+        visualEffect = UIGlassEffect()
+      } else {
+        visualEffect = UIBlurEffect(style: tint.toBlurEffect())
+      }
     }
   }
 
diff --git a/node_modules/expo-blur/ios/TintStyle.swift b/node_modules/expo-blur/ios/TintStyle.swift
index ca79ed4..241567d 100644
--- a/node_modules/expo-blur/ios/TintStyle.swift
+++ b/node_modules/expo-blur/ios/TintStyle.swift
@@ -1,6 +1,7 @@
 import ExpoModulesCore
 
 public enum TintStyle: String, Enumerable {
+  case glass
   case `default`
   case extraLight
   case light
@@ -26,6 +27,8 @@ public enum TintStyle: String, Enumerable {
   func toBlurEffect() -> UIBlurEffect.Style {
 #if os(tvOS)
     switch self {
+    case .glass:
+      return .regular
     case .default:
       return .regular
     case .extraLight:
@@ -71,6 +74,8 @@ public enum TintStyle: String, Enumerable {
     }
 #else
     switch self {
+    case .glass:
+      return .systemUltraThinMaterial
     case .default:
       return .regular
     case .extraLight:
