diff --git a/node_modules/react-native-screens/ios/RNSModule.mm b/node_modules/react-native-screens/ios/RNSModule.mm
index f3f14d3..d6c0ec3 100644
--- a/node_modules/react-native-screens/ios/RNSModule.mm
+++ b/node_modules/react-native-screens/ios/RNSModule.mm
@@ -1,3 +1,4 @@
+#include <utility>
 #import "RNSModule.h"
 #import <React/RCTBridge+Private.h>
 #import <React/RCTBridge.h>
diff --git a/node_modules/react-native-screens/ios/RNSScreen.mm b/node_modules/react-native-screens/ios/RNSScreen.mm
index 6d05957..1ade4b9 100644
--- a/node_modules/react-native-screens/ios/RNSScreen.mm
+++ b/node_modules/react-native-screens/ios/RNSScreen.mm
@@ -1,3 +1,4 @@
+#include <utility>
 #import <UIKit/UIKit.h>
 
 #import "RNSScreen.h"
diff --git a/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm b/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
index bda667a..da52aa4 100644
--- a/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
+++ b/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
@@ -1,3 +1,4 @@
+#include <utility>
 #ifdef RCT_NEW_ARCH_ENABLED
 #import <React/RCTConversions.h>
 #import <React/RCTFabricComponentsPlugins.h>
